packages/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -280,6 +285,7 @@ export async function assertIdentityTransformation(
         // known cases for the prop expecting to have been changed by the transformation under normal circumstances
         // - federation guid will be generated if it didn't exist
         // - jsonProperties may include remapped ids
+        // FIXME<Mike>: Doesn't this allow propChanges to ANY props if sourceElem.federationGuid is undefined?
         const propChangesAllowed = allowPropChange?.(sourceElem, targetElem, propName)
           ?? (sourceElem.federationGuid === undefined || propName === "jsonProperties");
```

Comment:
you can remove the FIXME after you change this to 
```suggestion
          ?? ((propName === "federationGuid" && sourceElem.federationGuid === undefined) || propName === "jsonProperties");
```

Reply:
done